psimitab-parser/src/main/java/psidev/psi/mi/tab/model/ParameterImpl.java

```diff
@@ -217,10 +219,13 @@ public void setValue(String value) {
                     case 3:
                         if (!matcher.group(3).isEmpty()) {
                             exponent = Integer.parseInt(matcher.group(3));
+                            userDefinedExponent = true;
                         }
                     case 2:
                         if (!matcher.group(2).isEmpty()) {
                             base = Integer.parseInt(matcher.group(2));
+                            userDefinedBase = true;
+                            if (base == 0) throw new IllegalArgumentException("No point in setting base to zero");
```

Comment:
Can we update the message to something like: "Zero is not allowed as a value for base" ?